### PR TITLE
Changes in Authorization API - “startTrial=false” parameter

### PR DESCRIPTION
**Pull request by olegrise**
@alexey-rise please review. Unfortunatelly the stage environment is not working correctly, so we can't really see the compiled version.

### CODE REVIEW

File: _posts/developer/store-api/store-auth/2014-10-01-store-start-trial-api.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title:  "Start Trial API"
+date:   2017-04-24 10:52:00
+category: developer/store-api/store-auth
+order: 4
+---
+
+You need to call Store Start Trial API if you want to start a trial outside of your widget. Normally you don't need to use this API because trial starts automatically when a widget calls Authorization API the first time. However when you call Store Authorization API with the `startTrial=false` parameter, the trial never starts. If you want to start a trial from the widget itself, you can either use Authorization API without `startTrial=false` parameter or call Start Trial API. If you want to start a trial outside of the widget, then it's better to use Start Trial API.
+
+
+##Store Authorization API Endpoint
```
Comment (alexey-rise) [inline _posts/developer/store-api/store-auth/2014-10-01-store-start-trial-api.md]: @olegrise should this be "Start Trial API Endpoint" instead?

File: _posts/developer/store-api/store-auth/2014-10-01-store-start-trial-api.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+`company_id` REQUIRED is the id of the Company for which trial should be started
+
+**API Response Parameters**
+
+`error` is an empty string if call was successful, otherwise the error description
+
+_ _ _
+
+
+**Example GET request**
```
Comment (alexey-rise) [inline _posts/developer/store-api/store-auth/2014-10-01-store-start-trial-api.md]: @olegrise I think we need another example without `callback`

File: _posts/developer/store-api/store-auth/2014-10-01-store-start-trial-api.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+category: developer/store-api/store-auth
+order: 4
+---
+
+You need to call Store Start Trial API if you want to start a trial outside of your widget. Normally you don't need to use this API because trial starts automatically when a widget calls Authorization API the first time. However when you call Store Authorization API with the `startTrial=false` parameter, the trial never starts. If you want to start a trial from the widget itself, you can either use Authorization API without `startTrial=false` parameter or call Start Trial API. If you want to start a trial outside of the widget, then it's better to use Start Trial API.
+
+
+##Store Authorization API Endpoint
+
+To call Store Authorization API, send the following GET request **over SSL**:
```
Comment (alexey-rise) [inline _posts/developer/store-api/store-auth/2014-10-01-store-start-trial-api.md]: @olegrise same point re "Start Trial API"

**Comment by olegrise** [comment]
@alexey-rise here is the stage URL https://help-stage.risevision.com/feature-authorization-api/_site/developer/store-api/store-auth/store-auth-api

**Comment by olegrise** [comment]
@alexey-rise please review PR fixes

**Comment by alexey-rise** [comment]
@olegrise looks good :+1:

**Pull request merged**